deploy/olm-catalog/kubevirt-hyperconverged/0.0.2/kubevirt-hyperconverged-operator.v0.0.2.clusterserviceversion.yaml

```diff
@@ -1402,7 +1420,7 @@ spec:
                         fieldPath: metadata.name
                   - name: OPERATOR_NAME
                     value: node-maintenance-operator
-                  image: quay.io/kubevirt/node-maintenance-operator:v0.3.0
+                  image: quay.io/kubevirt/node-maintenance-operator:v0.4.0
```

Comment:
yeah. this is strange I wouldn't expect your stuff to be messing with this.

Reply:
Guessing the same as above. Possible someone performed and update and didn't run build-manifest.sh.